src/main/resources/com/mysql/cj/LocalizedErrorMessages.properties

```diff
@@ -1099,4 +1099,7 @@ ClusterAwareReaderFailoverHandler.2=[ClusterAwareReaderFailoverHandler] Connecte
 ClusterAwareReaderFailoverHandler.3=[ClusterAwareReaderFailoverHandler] Trying to connect to reader [{0,number,#}] ''{1}''
 ClusterAwareReaderFailoverHandler.4=[ClusterAwareReaderFailoverHandler] Connected to reader [{0,number,#}] ''{1}''
 ClusterAwareReaderFailoverHandler.5=[ClusterAwareReaderFailoverHandler] Failed to connect to reader [{0,number,#}] ''{1}''
-ClusterAwareReaderFailoverHandler.6=[ClusterAwareReaderFailoverHandler] {0} was called with an invalid (null or empty) topology
\ No newline at end of file
+ClusterAwareReaderFailoverHandler.6=[ClusterAwareReaderFailoverHandler] {0} was called with an invalid (null or empty) topology
+
+AWSSecretsManagerPlugin.1=[AWSSecretsManagerPlugin] Required dependencies 'com.amazonaws.services.secretsmanager.AWSSecretsManager' and/or 'com.amazonaws.secretsmanager.caching.SecretCache' are not on the classpath
```

Comment:
I'd replace exact class names with dependent library names

Reply:
Similar comment about plug to be extracted to another repo, this resource file won't be available for modifications.